slides/module8.md

```diff
@@ -28,3 +32,213 @@ add the skills the trainee will obtain
 specify the agenda
 
 ------
+
+**Prerequisite**: [Typeclass pattern]
+
+[Typeclass pattern]: https://www.youtube.com/watch?v=sVMES4RZF-8
```

Comment:
I'd suggest as a prerequisite Scala's Implicits instead of this video which assumes that the reader, of this specific module, knows about already.

Reply:
Addressed.